extensions/roc-plugin-repo/src/index.js

```diff
@@ -302,9 +308,28 @@ module.exports.roc = {
             description: 'If projects should be published',
           },
           tag: {
-            validator: validators.isString,
-            default: 'latest',
-            description: 'dist-tag to be used when publishing',
+            default: true,
+            validator: validators.oneOf(
+              validators.isBoolean,
+              validators.isString,
+            ),
+            description:
+              'If a git tag should be created and what it should be called. For non-monorepos the default will be the version and the date otherwise.',
```

Comment:
`otherwise` what? I think you meant `'For non-monorepos the default will be the version and the date.'`